src/app/zwiazek/kalendarz/_components/Events/Card.tsx

```diff
@@ -0,0 +1,51 @@
+import Image from "next/image";
+import { Heading } from "@/components/shared/Heading/Heading";
+
+interface CardProps {
+  title: string;
+  image: {
+    src: string;
+    alt: string;
+  };
+  date: string;
+  description: string;
+  isLast: boolean;
```

Comment:
Not a fan of using JS to indicate last (maybe we could somehow use CSS or just proper composition while mapping) but we can leave it as it is